test/steps/kogitoapp.go

```diff
@@ -371,3 +371,15 @@ func parseKogitoAppKafkaRow(row *TableRow, kogitoApp *framework.KogitoAppHolder,
 		*profilesPtr = append(*profilesPtr, "events")
 	}
 }
+
+// mustParseEnabledDisabled parse a boolean string value
+func mustParseEnabledDisabled(value string) bool {
```

Comment:
Couldn't you reuse `mappers.mustParseEnabledDisabled` ?

Reply:
Done.